src/webui/www/public/statistics.html

```diff
@@ -1,4 +1,4 @@
-<h3>User Statistics</h3>
+<h3>QBT_TR(User Statistics)QBT_TR[CONTEXT=StatsDialog]</h3>
```

Comment:
>the string comes back untranslated

the translated string is for `User statistics` (small s in statistics)

As for the other issue i didn't try `--profile` but for me it takes a variable amount of time every time i try it to fetch the statistics. From less than a second to more than 10.